config/crd/bases/operator.kyma-project.io_thirdparties.yaml

```diff
@@ -0,0 +1,40 @@
+---
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
this crd should not persist in here, because it will persist in the final raw manifest, we don't want to third party CRD become module managed resources.